### PR TITLE
app-misc/gcal: 4.1 version bump

### PR DESCRIPTION
**Pull request by asartori86**
This is my first PR for an ebuild. not sure if I did it the right way..

### CODE REVIEW

File: app-misc/gcal/gcal-4.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+inherit eutils flag-o-matic toolchain-funcs
+
+DESCRIPTION="The GNU Calendar - a replacement for cal"
+HOMEPAGE="https://www.gnu.org/software/gcal/"
+SRC_URI="mirror://gnu/gcal/${P}.tar.xz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="amd64 ~arm ppc x86 ~amd64-linux ~x86-linux ~ppc-macos ~x86-macos"
```
[the remark this answers is not in the document]
Comment (titanofold) [reply app-misc/gcal/gcal-4.1.ebuild]: There's a tool for this:

     ekeyword ~all gcal-4.1.ebuild

We also have vim and Emacs modes that make this easy, too.

[the remark this answers is not in the document]
Comment (asartori86) [reply app-misc/gcal/gcal-4.1.ebuild]: thanks @gyakovlev  I have some issues with repoman..
Probably I missed something.

I cloned this repo in `~/opt/src/gentoo` (while emerge still watches the one in `/usr/portage`). When I run 
```
 $ repoman scan

RepoMan scours the neighborhood...

*** the local copy of metadata.xsd needs to be refetched, doing that now


Repoman: Need user access
```

what does this mean?

**Comment by asartori86** [comment]
thanks @monsieurp, I would be glad to fix the above issue.. do you have any hint on how to make repoman work?

**Comment by monsieurp** [comment]
Hey @asartori86. How do you mean? It should work out of the box. Emerge it and there you have it.

**Comment by asartori86** [comment]
@monsieurp, this issue :)
> I cloned this repo in ~/opt/src/gentoo (while emerge still watches the one in /usr/portage). When I run
> 
>  $ repoman scan
> 
> RepoMan scours the neighborhood...
> 
> *** the local copy of metadata.xsd needs to be refetched, doing that now
> 
> 
> Repoman: Need user access
> 
> what does this mean?

**Comment by jaakristioja** [comment]
> Repoman: Need user access
> 
> what does this mean?

@asartori86 See Gentoo bugs [#574130](https://bugs.gentoo.org/574130) and [#596670](https://bugs.gentoo.org/596670).

**Pull request closed without merging**